Cargo.toml

```diff
@@ -7,16 +7,20 @@ edition = "2018"
 [features]
 # Enable this feature to turn on additional GC optimizations that are only
 # possible with the rustgc fork of the compiler.
-rustgc = []
+standalone = ["allocator"]
 
 # Enable various GC based statistics. Stats are disabled by default as they have
 # a run-time cost and are expected to only be used for profiling purposes.
 gc_stats = []
 
 [dependencies]
 libc = "*"
+allocator = { path = "./allocator", optional = true }
```

Comment:
Is the `./` intentional and/or necessary?

Reply:
I suppose not. I copied this style from the Rust docs when I reminded myself about how to use workspaces. I can change it if you like.

Reply:
I guess it's partly a naive question on my part about what it means to have a sub-crate that's an immediate sub-directory. I haven't really looked at workspaces in any detail, so I don't have a meaningful opinion on this.

Reply:
Since they both seem to work, I've changed it to the more familiar path style that we're used to: 299fb96